### DISCUSSION TITLE
posts/2022-01-10-scraping-used-boats/

### DISCUSSION THREAD

**Discussion by giscus** [L-Groeninger/lukas-r.blog#39] [Comments]
# posts/2022-01-10-scraping-used-boats/

How to find the perfect sailing yacht.

https://lukas-r.blog/posts/2022-01-10-scraping-used-boats/

**Comment by devbastic** [comment]
Very interesting, Lukas. Going to check this out in detail. Thanks!

**Reply by L-Groeninger** [reply]
That's nice to hear, thanks!:)

**Comment by saschner** [comment]
This is incredible. I'm going to be scaping Florida boats and had no idea that anyone else had figured this out before me.  It's amazing.

**Reply by L-Groeninger** [reply]
Thanks a lot:)
